### PR TITLE
Hide editing feature #161

### PR DESCRIPTION
**Pull request by emymaria**
Disabling editing when enable editing is unchecked in settings

### CODE REVIEW

File: src/app/components/generics/link-input/link-input.component.ts [hunk 1]
```diff
@@ -13,9 +18,14 @@ export class LinkInputComponent implements OnInit {
   @Output() disable = new EventEmitter<void>();
   linkSearchText = '';
 
-  constructor() {}
+  readonly UiFeatures = UiFeatures;
```
Comment (lharzenetter) [inline src/app/components/generics/link-input/link-input.component.ts:21]: I guess you don't need this here.

File: src/app/components/generics/property-input/checkbox-input.component.ts [hunk 2]
```diff
@@ -13,6 +18,15 @@ export class CheckboxInputComponent {
   @Input() value: boolean;
   @Input() baseValue: boolean;
 
+  readonly UiFeatures = UiFeatures;
```
Comment (lharzenetter) [inline src/app/components/generics/property-input/checkbox-input.component.ts:21]: I guess you don't need this here.

File: src/app/components/generics/property-input/text-input.component.ts [hunk 3]
```diff
@@ -41,10 +46,14 @@ export class TextInputComponent implements OnInit, OnChanges {
   packedLatexValue: string;
   urlToRenderedBlob: SafeUrl;
 
+  readonly UiFeatures = UiFeatures;
```
Comment (lharzenetter) [inline src/app/components/generics/property-input/text-input.component.ts:49]: I guess you don't need this here.

**Review by salmma — CHANGES_REQUESTED** [review]
As discussed, please 
* hide the "add item" and "delete item" buttons in all lists
* hide the Actions column in all lists
* hide the edit and delete buttons, e.g., for given "Required compute resource properties" in the "General" tab of an algorithm/implementation/compute resource.
* hide the add button for "Required compute resource properties"
* hide the delete button for given Problem types in the "General" tab of an algorithm
* hide the delete button for given Application areas in the "General" tab of an algorithm

**Review by salmma — APPROVED** [review]
LGTM

**Review by lharzenetter — APPROVED** [review]

**Pull request merged**